src/components/government.js

```diff
@@ -0,0 +1,27 @@
+import { useEffect, useState } from 'react';
+
+function Government() {
+  const url = 'https://api.api-ninjas.com/v1/quotes?category=government';
+  const [content, setContent] = useState(<p>Loading ...</p>);
+  useEffect(() => {
+    const fetchData = async () => {
+      try {
+        const request = await fetch(url, {
+          method: 'GET',
```

Comment:
[OPTIONAL] Although this logic is working, but for separation of concerns, I strongly recommend using multiple states or a state object for the error, loading and when the data is retrieved instead using just one state value to manage them. You will often see this pattern used largely in the wild.